tests/configs/gpii.tests.untrusted.production.config.json5

```diff
@@ -0,0 +1,15 @@
+{
+    "type": "gpii.tests.untrusted.production.config",
+    "options": {
+        "mainServerPort": 9082,
+        "distributeOptions": {
```

Comment:
This "distributeOptions" block is no longer needed with the merge of [the pull request for GPII-3424](https://github.com/GPII/universal/pull/722).